### DISCUSSION TITLE
Windowed Startup

### DISCUSSION THREAD

**Discussion by aknight2015** [GitSquared/edex-ui#1133] [Questions] [answered]
For some reason it starts in windowed mode. As far as I know I didn't do anything to tell it to start windowed. How can I fix that?

**Comment by GitSquared** [comment]
What OS?

**Comment by aknight2015** [comment]
Whoops. I'm so sorry. I guess that's kind of important. I'm using
Windows 10 64-bit. Also, I LOVE THE PROGRAM. It's an awesome little
toy.

**Reply by GitSquared** [reply]
Cheers ;)

You're using the latest version and haven't modified any settings then?
Could you try deleting the folder at `%AppData%\eDEX-UI` and see if it works?

**Comment by aknight2015** [comment, marked as answer]
That seems to have done it.

**Reply by GitSquared** [reply]
Great!
